### ISSUE TITLE
Comments on JOSS paper document

### ISSUE THREAD

**Issue by ymahlau** [Jan-Williams/OpenReservoirComputing#62]
- Lukoševičius & Jaeger, 2009 is a very old paper before the rise of modern AI training on GPU. This makes it likely a poor reference to justify comparison to modern artificial neural networks.
- Text in Fig. 1 extremely small
- The three-lines-of-code claim appears twice (lines 34–35, 39) but no code snippet is given
- Pytorch-ESN is dismissed quickly, a one-liner on what features are not supported would be helpful
- Continuous time in table 1 is a bit misleading, for example ReservoirComputing.jl seems to support continous dynamics via a fixed time step integration. This is of course something very different from the `diffrax` approach by ORC, but confused me at the beginning. A short clarification on that would be helpful in the paper.

Typographic comments on the paper (generated by AI):
- Checkmarks and Xs are not rendered correctly in the description of Table 1
- Missing Period: On page 3 (line 100), there is a missing period after the inline equation for $y_{t}$. It currently reads: "...approximation of some low-dimensional signal $y_{t}$ Depending on the task at hand...". It should be amended to: "...approximation of some low-dimensional signal $y_{t}$. Depending on the task at hand..."

**Comment by ymahlau** [comment]
https://github.com/openjournals/joss-reviews/issues/10486

**Comment by Jan-Williams** [comment]
> * Lukoševičius & Jaeger, 2009 is a very old paper before the rise of modern AI training on GPU. This makes it likely a poor reference to justify comparison to modern artificial neural networks.

I have included additional references for the associated claim. Lukoševičius & Jaeger make sense to include still as reason for the speed up (Ridge Regression vs backdrop and gradient descent) still holds, despite the paper being old.

> * Text in Fig. 1 extremely small

I've modified the diagram to be more legible.

> * The three-lines-of-code claim appears twice (lines 34–35, 39) but no code snippet is given

The most recent draft of the paper includes the code snippet.

> * Pytorch-ESN is dismissed quickly, a one-liner on what features are not supported would be helpful

I have added the following language to the paper to address this:
"For instance, it does not support sparse matrix operations nor parallel reservoirs."

> * Continuous time in table 1 is a bit misleading, for example ReservoirComputing.jl seems to support continous dynamics via a fixed time step integration. This is of course something very different from the `diffrax` approach by ORC, but confused me at the beginning. A short clarification on that would be helpful in the paper.

I'm open to changing this if your concerns still hold, but [this issue](https://github.com/SciML/ReservoirComputing.jl/issues/397) raised by the primary author of ReservoirComputing.jl makes me think that the current status of the table is fair.

Thank you for the typographic comments, they've been addressed. 

I'll leave this issue open for now in case you have additional thoughts/comments.

**Comment by ymahlau** [comment]
Sounds all good to me!
